test/utils/math/Math.t.sol

```diff
@@ -308,6 +308,27 @@ contract MathTest is Test {
         }
     }
 
+    // REVERSE BITS
+    function testSymbolicReverseBitsUint256(uint256 value) public pure {
+        assertEq(Math.reverseBitsUint256(Math.reverseBitsUint256(value)), value);
+    }
+
+    function testSymbolicReverseBitsUint128(uint128 value) public pure {
+        assertEq(Math.reverseBitsUint128(uint128(Math.reverseBitsUint128(value))), value);
```

Comment:
Ideally, we would also inject "dirty upper bits"

- get the value from fuzzing (clean)
- turn it into dirtyValue 
- check that assertEq(Math.reverseBitsUint128(Math.reverseBitsUint128(dirtyValue)), value);